### PR TITLE
Add script to generate file with all required exports.

### PR DESCRIPTION
**Pull request by mkudlej**
User can load the file with exports and run integration tests. User should be login into OCP and project with 3scale deployment should be set.

### CODE REVIEW

**Review by jsmolar — APPROVED** [review]

File: get_exports.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
```
Comment (mganisin) [inline get_exports.sh:1]: FYI there is a job in jenkins called 3scale-api-python-tests to run integration tests. Obviously it does something similar, it has to to run the tests. It just doesn't create anything, only collecting information from existing deployment and objects.

File: get_exports.sh [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+# namespace with 3scale
+
+NAMESPACE="$(oc project -q)"
+if [ $# -gt 0 ]; then
+  NAMESPACE=$1
+fi
+
+# Tenant
+
+SEC_NAME="threescale-provider-account-${RANDOM}"
```
Comment (pehala) [inline get_exports.sh:15]: What is SEC_NAME?

**Review by pehala — CHANGES_REQUESTED** [review]
I am not convinced we need this. For the basic test run, we should only need a few variables, and as far as I know, there is no need to create anything on 3scale for tests to run.
If we want a script that would collect data from deployed 3scale, I would much rather see something that just collects data from secrets and tools.

File: get_exports.sh (hunk 1 above)
Comment (mkudlej) [reply get_exports.sh:1]: This is helping script for manual run of unit tests.

**Comment by mkudlej** [comment]
After talk with @pehala I've decided to close this PR without merging it. If anybody would like to us this script, just let me know.

**Pull request closed without merging**